src/components/block/block.stories.ts

```diff
@@ -174,12 +174,6 @@ export const disabled_TestOnly = (): string => html`<calcite-block
   </calcite-block-section>
 </calcite-block>`;
 
-export const paddingDisabled_TestOnly = (): string => html` <calcite-panel heading="Properties">
-  <calcite-block heading="Example block heading" summary="example summary heading" collapsible open disable-padding>
-    <div>calcite components ninja</div>
-  </calcite-block>
-</calcite-panel>`;
```

Comment:
Should we replace this with a snapshot setting `--calcite-block-padding` to zero, instead of completely losing coverage?

Reply:
yeah ill put it back. 👍